src/Compute/Compute/ChangeLog.md

```diff
@@ -37,6 +41,12 @@
 * Added new cmdlets:
     - Add-AzVmssRunCommand
     - Remove-AzVmssRunCommand
+    - Set-AzVMRunCommand
+    - Get-AzVMRunCommand
+    - Remove-AzVMRunCommand
+    - Set-AzVmssVMRunCommand
+    - Remove-AzVmssVMRunCommand
+    - Get-AzVmssVMRunCommand
```

Comment:
These are already on main branch. It's unlikely to cause any problem but I'll double check after merge

Reply:
took this out to not be making changes to older version changelog